### ISSUE TITLE
Add auto costing options similar to bicycle to support electric cars, scooters, etc.

### ISSUE THREAD

**Issue by rmglennon** [valhalla/valhalla#912]
This request came in through support (see Desk 932). 

The suggestion is to enhance auto costing to include options that are similar to bicycle. For example, for electric vehicles and scooters (or trucks, too), you would want to minimize uphills. You can apparently emulate scooter directions with bicycle costing for "use_roads", "use_hills", cycling_speed", but this should be part of auto costing.

Examples: 
- Vehicles' max speed is 30-35mph, so no roads with a speed limit of 40mph or more, and no highways.
- Small or electric engines do not thrive on steep hills.

Basically, add more options for auto similar to the priorities for bicycle and pedestrian. 

cc @dnesbitt61 @kevinkreiser @acwilton

**Comment by mwujek** [comment]
Thank you for adding my request @rmglennon! I'm happy to add more color to this request if needed—just tag me

**Comment by dnesbitt61** [comment]
Initial capability has been deployed for motor_scooter (not part of auto costing, a new costing method). Please open other issues if there are specific problems or proposed changes.
